server/test/unit/server/test_logs.py

```diff
@@ -765,8 +763,7 @@ def fake_getLogger(name=None):
         # We should have used the user's setting
         root_logger.setLevel.assert_called_once_with(logging.ERROR)
 
-    @mock.patch('pulp.server.logs.config.config.get',
-                side_effect=ConfigParser.NoOptionError('server', 'log_level'))
+    @mock.patch('pulp.server.logs.config.config.get', return_value='INFO')
```

Comment:
Yeah I tried to structure this test in such a way that I wasn't testing `get('server', 'log_level')` but only the `get('server', 'log_type')` option since the `log_level` is covered in another test. It seems like the unit tests mock a lot so many parts of the config get duplicated to get around that.